consensus/types.go

```diff
@@ -51,7 +67,7 @@ type BlockHashValue = [32]byte
 type FastHotStuffEventLoop interface {
 	GetEvents() chan *FastHotStuffEvent
 
-	Init(time.Duration, time.Duration, BlockWithValidatorList, []BlockWithValidatorList) error
+	Init(time.Duration, time.Duration, QuorumCertificate, BlockWithValidatorList, []BlockWithValidatorList) error
 	GetCurrentView() uint64
 	AdvanceViewOnTimeout() (uint64, error)
 	ProcessTipBlock(BlockWithValidatorList, []BlockWithValidatorList) error
```

Comment:
Every time I hit "Go to Definition" for one of these, it takes me here rather than to the implementation. Even hitting "Show Usages" doesn't take me there. Is there an easy way to jump directly to the implementation that I'm not aware of?

Reply:
The event loop is referenced as the interface type. So, no there isn't a a built-in way to go to the implementation directly in VS Code. You'd have to Ctrl+Shift+F to search for it.

Reply:
The interface is useful so we can test the consensus using a mock event loop type. Marking this as resolved since it's not a code issue, and is instead an IDE limitation